Sanitize SQL generated by SqlServerSequenceValueGenerator

mikary opened the pull request:
Using the ```ISqlServerSqlGenerator``` to sanitize the sequence name.
@ajcvickers @bricelam

bricelam commented:
:shipit:

anpete commented on src/EntityFramework.SqlServer/SqlServerSequenceValueGenerator.cs:
```diff
@@ -13,28 +13,30 @@ namespace Microsoft.Data.Entity.SqlServer
         {
-            // TODO: Parameterize query and/or delimit identifier without using SqlServerMigrationOperationSqlGenerator
-            var sql = string.Format(CultureInfo.InvariantCulture, "SELECT NEXT VALUE FOR {0}", _sequenceName);
+            var sql = "SELECT NEXT VALUE FOR " + _sqlGenerator.DelimitIdentifier(_sequenceName);
```
Can this be added to the SQL generator so that it can be tested?

mikary replied:
Are you thinking something like ```AppendSelectValue(StringBuilder)```?

anpete replied:
"Generate" is probably enough here, but yes, just so you can write a test for the change.

anpete commented:
Doesn't look like there is a test for this change?

mikary commented:
Updated based on PR feedback, I added the method to the SqlGenerator, but the name I came up with, ``` GenerateNextSequenceValueOperation``` seems a bit too verbose to me.
